### DISCUSSION TITLE
Stuck at 99.9% #105 Ryuk

### DISCUSSION THREAD

**Discussion by ms-saltation** [cssbattle/forum#342] [Q&A] [answered]
Not able to get 100%, anyone has a hint ?
Is it the upper triangle shape or something with the eyes ?

`<p><dl><dd><a><style>*+*{background:#BAC7CE}p{background:#475862;height:200;rotate:45deg;margin:-196 76 0 0}dd{width:120;height:120;background:#000;border-radius:0 60px;margin:118 22;rotate:-15deg}dl{width:100;-webkit-box-reflect:right 184px}a{box-shadow:inset -4px 14px#5A6042;padding:50;position:fixed;border-radius:50%;margin:10;background:radial-gradient(25vw,#4E2B24 15px,0,#000 5vw,0,#868A64
`

**Comment by Beowolve** [comment, marked as answer]
I think I have never found a working box-shadow based version for this. I suggest going with radial-gradient on this target, that works.

**Reply by rubenbelem** [reply]
I've just come across these techniques like using box-shadow or radial-gradient by checking some top solutions, and I am really impressed and curious about it hehe. Is there any tutorial, blog post, video etc. you guys recommend to get a better grasp on it?

**Reply by Beowolve** [reply]
I am actually working on something, not ready yet, but box-shadows will be covered for sure.

**Reply by rubenbelem** [reply]
Awesome! Do you have like a Medium blog or YouTube channel where you plan to post it?

**Reply by Beowolve** [reply]
Not yet, I will post something when its ready

**Reply by rubenbelem** [reply]
Nice!

Btw, I got a 100% match using box-shadow, like this:
`
<eb></eb><le></le><eb></eb><le></le><fr></fr><style>body{background:#bac7ce;display:grid;place-items:center}body>*{position:absolute}le{width:100px;height:100px;border-radius:50%;background:#4e2b24;transform:translate(-110px,40px);box-shadow:inset 0 15px 0 0 #5a6042,inset 0 0 0 30px #868a64,inset 0 0 0 35px #000}eb{width:120px;height:120px;background:#000;border-top-right-radius:60px;border-bottom-left-radius:60px;rotate:-15deg;left:30px;bottom:50px}le:last-of-type{transform:translate(110px,40px)}eb:last-of-type{rotate:-75deg;left:250px;bottom:50px}fr{border:91px solid transparent;border-top-color:#475862;top:0}
`
